matrix/lambdas/api/core.py

```diff
@@ -55,11 +59,31 @@ def post_matrix(body: dict):
             }, requests.codes.accepted
 
 
+# TODO: write tests
 def get_matrix(request_id: str):
+    dynamo_handler = DynamoHandler()
+    # TODO: error handling: invalid request_id
+    ready, ready = dynamo_handler.increment_table_field(DynamoTable.STATE_TABLE,
```

Comment:
Why are you incrementing the reducer executions in the get_matrix request?

Reply:
Hacky way to read from the table copied from port over. Will rewrite